crates/services/importer/src/ports.rs

```diff
@@ -32,21 +35,22 @@ pub trait Executor: Send + Sync {
 /// The database port used by the block importer.
 pub trait ImporterDatabase {
     /// Returns the latest block height.
-    fn latest_block_height(&self) -> StorageResult<BlockHeight>;
+    fn latest_block_height(&self) -> StorageResult<Option<BlockHeight>>;
```

Comment:
I'm confused by the cardinality of this. In which scenario would there be `None` block height? Won't we always have at least a genesis block?

I'm guessing some change upstream prompted you to change this, but from the context of an interface, the types are confusing to the consumer for ^^^ reason.

Reply:
The block importer is already responsible for committing the genesis block. Before, we expected that the underlying database would not be initialized while the database transaction would be. Now, we expect that both don't have a genesis block. The `Option` is better than the old way of checking the error.

<img width="1339" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/15c76e8e-edea-4687-9774-f916224f0360">

Reply:
Okay. I see now. Then I agree.